exporter/instanaexporter/internal/converter/span_converter_test.go

```diff
@@ -255,13 +255,17 @@ func TestSpanWithError(t *testing.T) {
 }
 
 func generateTraceID() (data [16]byte) {
-	rand.Read(data[:])
-
+	_, err := rand.Read(data[:])
+	if err != nil {
```

Comment:
might be best to return the error and use `require.NoError` in the caller. what do you think?